homeworks/roman.strokan_Strokan/homework_3/z2.js

```diff
@@ -0,0 +1,20 @@
+let a;
```

Comment:
why not putting `a` variable in `mainFunc` body? you put it the global scope without a need, should be fixed